### PR TITLE
Feature/check if spotify app is active android

### PR DESCRIPTION
**Pull request by brim-borium**
This cannot merged before #105

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -560,6 +612,22 @@ class _HomeState extends State<Home> {
...
+              : 'Spotify app connection is not active (currently not playing)'));
+
+      ScaffoldMessenger.of(context).showSnackBar(snackBar);
+    } on PlatformException catch (e) {
+      setStatus(e.code, message: e.message);
+    } on MissingPluginException {
+      setStatus('not implemented');
+    }
+  }
+
```
Comment (fotiDim) [inline example/lib/main.dart:630]: Kudos for the addition!

**Comment by fotiDim** [comment]
Looks good! Just a couple of conflicts that need to be resolved.
Shall we also add an entry in the changelog for the upcoming release?

File: example/lib/main.dart (hunk 1 above)
Comment (fotiDim) [reply example/lib/main.dart:630]: Come to think of it I remember there is some other functionality that is not demoed in the example app. Do we want to address it or leave things as they are? Even the official Spotify sample app does not cover their entire API.

**Comment by brim-borium** [comment]
I will update the changeling on the main branch when we release, I think there are a couple of changes that are not in there yet

File: example/lib/main.dart (hunk 1 above)
Comment (brim-borium) [reply example/lib/main.dart:630]: Do you know which one that is? I could add it in this pr

Comment (fotiDim) [reply example/lib/main.dart:630]: With a quick search I spotted:
subscribeUserStatus
subscribeCapabilities
getLibraryState
getCapabilities
removeFromLibrary
skipToIndex

which are missing from the example app.

**Review by fotiDim — APPROVED** [review]

**Comment by fotiDim** [comment]
Hmm I tried to run the app on Android (on master) and I had to change the `distributionUrl` in `android/gradle/wrapper/gradle-wrapper.properties`. Should that be updated as well in the repo? Currently it is `distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip`.

File: example/lib/main.dart (hunk 1 above)
Comment (brim-borium) [reply example/lib/main.dart:630]: Lets take these to another pr and implement them, if possible

**Comment by brim-borium** [comment]
> Hmm I tried to run the app on Android (on master) and I had to change the `distributionUrl` in `android/gradle/wrapper/gradle-wrapper.properties`. Should that be updated as well in the repo? Currently it is `distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip`.

@fotiDim this could be coming from this pr #104. Did this crash on your site if you didn't update the url?

**Comment by fotiDim** [comment]
> > Hmm I tried to run the app on Android (on master) and I had to change the `distributionUrl` in `android/gradle/wrapper/gradle-wrapper.properties`. Should that be updated as well in the repo? Currently it is `distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip`.
> 
> @fotiDim this could be coming from this pr #104. Did this crash on your site if you didn't update the url? And what were the changes you had to make?

It was not a crash. I was unable to build:
```
Minimum supported Gradle version is 6.1.1. Current version is 5.6.4.
```
I had to change `distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip` to `distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip`

File: example/lib/main.dart (hunk 1 above)
Comment (fotiDim) [reply example/lib/main.dart:630]: I can open an issue and implement some time in the future. I don't see that as urgent.

Comment (brim-borium) [reply example/lib/main.dart:630]: > I can open an issue and we can implement them some time in the future. I don't see that as urgent.

Yeah good idea

**Comment by brim-borium** [comment]
> > > Hmm I tried to run the app on Android (on master) and I had to change the `distributionUrl` in `android/gradle/wrapper/gradle-wrapper.properties`. Should that be updated as well in the repo? Currently it is `distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip`.
> > 
> > 
> > @fotiDim this could be coming from this pr #104. Did this crash on your site if you didn't update the url? And what were the changes you had to make?
> 
> It was not a crash. I was unable to build:
> 
> ```
> Minimum supported Gradle version is 6.1.1. Current version is 5.6.4.
> ```
> 
> I had to change `distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip` to `distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip`

Does this result in a breaking change for the package? Or was this just some local issue?

**Comment by fotiDim** [comment]
I am getting it on a fresh checkout of the repo. 
At the same time if I create a new flutter project then it is `distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip`.
My Android experience is not that great but it looks to me that we should set it to 6.7.

**Comment by fotiDim** [comment]
@brim-borium shall we merge it as is and create another PR before the next release in case the issue persists?

**Pull request merged**